media/css/externalpages/pocket/components/get-inspired.scss

```
@@ -0,0 +1,120 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+@import '~@mozilla-protocol/core/protocol/css/components/split';
+
+@import '../utils/variables';
+@import '../includes/nav';
+@import '../includes/footer';
+
+$image-path: '/media/img/externalpages/pocket';
+
+.section-heading {
+    @include text-title-sm;
+    font-family: $font-sans-alt;
+    font-weight: 500;
+}
+
+.sub-section-heading {
+    @include text-title-xs;
+    font-family: $font-sans-alt;
+    font-weight: 500;
+}
+
+.section-lede {
+    @include text-body-xl;
```

nit: this seems a bit big to me compared to other pages and the live site (21px to 16px)
```suggestion
    @include text-body-md;
```